src/Gui/ProjectBrowserService.cs

```diff
@@ -145,6 +145,10 @@ void TypeLibraries_CollectionChanged(object sender, NotifyCollectionChangedEvent
             {
             case NotifyCollectionChangedAction.Add:
                 AddComponents(e.NewItems);
+                foreach (var item in e.NewItems)
+                {
+                    SelectedObject = item;
```

Comment:
I don't understand the purpose of setting the `SelectedObject` in a loop?

Reply:
I want to select (and so show text editor) added script. Do you prefer
```
SelectedObject = e.NewItems.OfType<object>().First();
```
?

Reply:
I'm seeing a loop opening (potentially) many documents and I get nervous. I'd rather this just open the first one.

Reply:
Done in ea77fd9